packages/itwinui-css/README.md

```diff
@@ -2,10 +2,10 @@
   <picture>
     <source
       media='(prefers-color-scheme: dark)'
-      srcset='https://itwinplatformcdn.azureedge.net/iTwinUI/iTwinUI-new-dark-logo.png'
+      srcset='https://itwin.github.io/iTwinUI/logo.svg' 
```

Comment:
I think this should be the dark variant.

```suggestion
      srcset='https://itwin.github.io/iTwinUI/logo-dark.svg' 
```

This fixes the logo on the dark theme:

|Old|New|
|-|-|
|<img width="453" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/552f2c1b-226a-441d-a940-ba5d41dd50ed">|<img width="448" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/45d29bdf-21e5-4d8e-938a-85eeb25978fc">|

Reply:
made that change!